master/custom/builders.py

```diff
@@ -116,6 +117,7 @@ def get_builders(settings):
         ("AMD64 Fedora Rawhide LTO", "cstratak-fedora-rawhide-x86_64", LTONonDebugUnixBuild, UNSTABLE),
         ("AMD64 Fedora Rawhide LTO + PGO", "cstratak-fedora-rawhide-x86_64", LTOPGONonDebugBuild, UNSTABLE),
         ("AMD64 Arch Linux TraceRefs", "pablogsal-arch-x86_64", UnixTraceRefsBuild, STABLE),
+        ("AMD64 Arch Linux VintageParser", "pablogsal-arch-x86_64", UnixVintageParserBuild, STABLE),
```

Comment:
Hum, it may be safer to start with UNSTABLE, and only switch it to STABLE once we checked that it's stable.

Reply:
> Hum, it may be safer to start with UNSTABLE, and only switch it to STABLE once we checked that it's stable.

Is the old parser, so it should be stable, no?